libs/components/project.json

```
@@ -3,6 +3,16 @@
   "sourceRoot": "libs/components/src",
   "projectType": "library",
   "targets": {
+    "gen": {
```

maybe more explicit?

```suggestion
    "generateMeta": {
```